packages/manager/apps/dedicated/client/app/dedicated/server/interfaces/interfaces.html

```diff
@@ -55,10 +55,9 @@
         >
         </ola-step-checker>
     </div>
-
     <div
         class="col-md-4 mb-5"
-        data-ng-if="$ctrl.ola.isAvailable() && $ctrl.ola.isActivated() && $ctrl.ola.isConfigured()"
+        data-ng-if="$ctrl.ola.isAvailable() && $ctrl.ola.isActivated()"
```

Comment:
**nitpick**:
The two `data-ng-if="$ctrl.ola.isAvailable()` tests could be done once at L44